packages/frontend-acceptance-tests/src/features/8 - Recurring Payments/1_adult_licence-12-months_immediate_start_recurring.feature

```diff
@@ -0,0 +1,150 @@
+@browser
+Feature: I want to buy a recurring adult annual fishing
+
+  Background: Buy an adult fishing licence
+    Given  I am at the start of the purchase journey
+
+  Scenario: Scenario 1 - 12 Month Adult licence selecting 2 rod sea trout licence - Immediate start - Enter contact details
```

Comment:
This scenario description is inaccurate – the journey below selects a salmon licence, but here it says "2 rod sea trout". This mismatch is the case for all of the new scenarios across the new files – some will say "salmon" and actually select a 2-rod coarse licence, etc. It's possible there are some other mismatches as well so think all of these will need to be checked and fixed.

Give me a nudge when this is done and I'll get back on the rest of the review.

Reply:
looks ike were incorrect for others fie I haven't edited as we, wi go through them